src/GRA/Annotations.cs

```diff
@@ -26,6 +26,7 @@ public static class Home
 
         public static class Info
         {
+            public const string AltTextExplanation = "How should this {0} be described to those who can’t see it?";
```

Comment:
I think this annotation is only on the MissionControl side of things? If so it can be removed.

Reply:
Resolved in commit 8bd1657408c9144ff99a4f76cfde7331fb6685ec